modules/aws-vpc/input.tf

```diff
@@ -34,7 +34,7 @@ variable "region" {
 
 variable "internal-zone" {
   type        = "string"
-  default     = "sighup.io"
+  default     = "staging.k8s.sighup.io"
```

Comment:
Do we really need a default? I would like to "force" people to properly choose an internal zone without relaying on some other third-party domains.

Reply:
Default is like “how to use”

Reply:
I do not agree on this point: documentation is "how to use".

In this case I don't think that there is a good default value because you risk ending up with a cluster with the "wrong" `internal-zone`.

I would take this chance to remove the default altogether.

What do you think @angelbarrera92 @phisco?

Reply:
Agree with @lnovara. no default value in this case is the best option. Otherwise you will end up with an unexpected behavior. As @lnovara explain, the correct place to put a description about how to use it is in the `description` field.

Reply:
default value removed!